### ISSUE TITLE
Add episode count on covers and Option to reduce grid and cover size and .....

### ISSUE THREAD

**Issue by shad90** [z-------------/CPod#156]
**CPod version:** 1.27.1

**Description:** 

1.   Please add option to reduce cover size. Covers are too large for my taste. 
2.   Episode count should be displayed on Cover.
3.   As asked earlier please add ability to open podcast details by just clicking the cover. There should be no need of Show Details button.
4.   Add ability to resize panels. I mean podcast cover grid panel is small. One should be able to resize it.

**Comment by nmsutton** [comment]
@shad90 I agree with all of these suggestions. Particularly point 4, which I was going to post here myself until I found you posted this. Many of us have several podcast subscriptions and the panel showing all podcasts listed is too small. The ability to resize that or just allow expanding it would be great.
